CNAME

```diff
@@ -1 +1 @@
-a11yproject.com
```

Comment:
I created a new pull request: https://github.com/a11yproject/a11yproject.com/pull/405
Delete this: https://github.com/a11yproject/a11yproject.com/pull/404